### PR TITLE
Task/idkjay/tlt 4605/add card to dashboard and permissions

### PR DESCRIPTION
**Pull request by idkjay**
- Added new permission to the new `lti_school_permissions` table in the QA and DEV postgres table for `course_info_v2` by mirroring the existing values currently set for `search_courses` permission. This way we can selectively roll out V2. They have not been put into prod and thus preventing the card from showing in prod as we work on the stories.
- Repositioned and renamed the existing `course_info_v2` card in the dashboard template. It was already there from a previous story.
- Some of this story's requirements were already met in previous stories, such as creating `PERMISSION_COURSE_INFO_V2` permission in `base.py`.
- Removed extra `LTI_SCHOOL_PERMISSIONS_TOOLS_PERMISSIONS` tuple from `base.py`.

SQL Command for creating new permissions records that mirror V1
```
INSERT INTO lti_school_permissions (permission, canvas_role, school_id)
SELECT 'course_info_v2', canvas_role, school_id
FROM lti_school_permissions
WHERE permission = 'search_courses';
```

Notes
- Deployed in QA
- Database changes committed in QA and DEV postgres

### CODE REVIEW

File: canvas_account_admin_tools/views.py [hunk 1]
```diff
@@ -183,6 +183,7 @@ def dashboard_account(request):
 
     return render(request, 'canvas_account_admin_tools/dashboard_account.html', {
         'search_courses_allowed': search_courses_allowed,
+        'course_info_v2_allowed': course_info_v2_allowed,
```
Comment (Chris-Thornton-Harvard) [inline canvas_account_admin_tools/views.py:186]: This duplicates line 196

**Comment by Chris-Thornton-Harvard** [comment]
We will want to keep the existing `search_courses` permission records and just add the new `course_info_v2` permissions that mirror V1 values. This way we can selectively roll out V2

**Comment by idkjay** [comment]
Added back all the `search_courses` V1 records and they exist alongside the V2 records.

**Comment by sapnamysore** [comment]
We willl want to have teh SQL recorded for using it for prod

**Comment by Chris-Thornton-Harvard** [comment]
👍

**Pull request merged**